single_level.py

```diff
@@ -27,14 +27,59 @@ def get_note_types(resource):
 		else:
 			note_types.append(note["jsonmodel_type"])
 	return note_types
+	
+def get_extent_number(resource):
+	extent_number = []
+	for extent in resource["extents"]:
+		if extent["number"] > 0:
+			extent_number.append("true")
+		else:
+			extent_number.append("false")
+	return extent_number
+	
+def get_language(resource):
+	d = resource
+	lang = []
+	if 'language' in d:
+		return "true"
+	else:
+		return "false"
 
+def get_creator(resource):
+	creator_type = []
+	for creator in resource["linked_agents"]:
+		if creator["role"] == ("creator"):
+			creator_type.append("true")
+		else:
+			creator_type.append("false")
+	return creator_type
+	
 def makeRow(resource):
 	global row
 	row = []
+	language_list = get_language(resource)
+	extent_list = get_extent_number(resource)
+	creator_list = get_creator(resource)
 	notes_list = get_note_types(resource)
 	row.append(resource["title"].encode("utf-8"))
 	row.append(resource["id_0"])
 	row.append(resource["publish"])
+			
+	if "true" in creator_list:
+		row.append("true")
+	else:
+		row.append("false")
+	
+	if "true" in language_list:
+		row.append("true")
+	else:
+		row.append("false")
+		
+	if "true" in extent_list:
```

Comment:
See my comment about the `creator_list` variable above - this seems awkward.
Part of my thinking is that eventually you'll want to return some of the actual content, so it would be helpful to have your functions return content rather than simply true or false. Make sense?